pkg/cfn/manager/iam.go

```diff
@@ -32,6 +32,7 @@ func (c *StackCollection) createIAMServiceAccountTask(errs chan error, spec *api
 	spec.Tags[api.IAMServiceAccountNameTag] = spec.NameString()
 
 	if err := c.CreateStack(name, stack, spec.Tags, nil, errs); err != nil {
+		logger.Info("to cleanup resources, run 'eksctl delete iamserviceaccount --region=%s --name=%s --namespace=%s'", c.spec.Metadata.Region, spec.Name, spec.Namespace)
```

Comment:
This message would appear before the actual error message, and I think it can be confusing. Can we have this printed either after the error message, or mention that an error occurred in this message itself?

```suggestion
		logger.Info("an error occurred creating the stack, to cleanup resources, run 'eksctl delete iamserviceaccount --region=%s --name=%s --namespace=%s'", c.spec.Metadata.Region, spec.Name, spec.Namespace)
```